.github/workflows/job_tensorflow_layer_tests.yml

```diff
@@ -0,0 +1,194 @@
+
+      - name: Setup Python ${{ env.PYTHON_VERSION }}
+        uses: ./openvino/.github/actions/setup_python
+        with:
+          version: ${{ env.PYTHON_VERSION }}
+          pip-cache-path: ${{ runner.os == 'Linux' && env.PIP_CACHE_PATH || '' }}
+          should-setup-pip-paths: ${{ runner.os == 'Linux' }}
+          self-hosted-runner: ${{ runner.os == 'Linux' }}
+
+      #
+      # Tests
+      #
+
+      - name: Install OpenVINO Python wheels
+        if: runner.os != 'Windows'
+        run: |
+          # Install the core OV wheel
+          python3 -m pip install ${INSTALL_DIR}/tools/openvino-*.whl
+
+          if [[ "${{ runner.arch }}" != "ARM64" ]]; then
+            python3 -m pip install ${INSTALL_DIR}/openvino_tokenizers-*.whl
+          fi
+
+          extras_to_install="caffe,kaldi,tensorflow2"
+
+          if [[ "${{ runner.arch }}" != "ARM64" ]]; then
+            extras_to_install="mxnet,$extras_to_install"
+          fi
+
+          # Find and install OV dev wheel
+          pushd ${INSTALL_DIR}/tools
+            ov_dev_wheel_name=$(find . -name 'openvino_dev*.whl')
+            python3 -m pip install $ov_dev_wheel_name[$extras_to_install]
+          popd
+
+      - name: Install OpenVINO Python wheels
+        if: runner.os == 'Windows'
+        run: |
+          # Find and install the core OV wheel
+          $ovCoreWheelPath=Get-ChildItem -Path "${{ env.INSTALL_DIR }}\tools" -Filter openvino-*.whl | % { $_.FullName }
+          python3 -m pip install "$ovCoreWheelPath"
+
+          # Find and install the core OV Tokenizers wheel
+          $ovCoreWheelPath=Get-ChildItem -Path "${{ env.INSTALL_DIR }}" -Filter openvino_tokenizers-*.whl | % { $_.FullName }
+          python3 -m pip install "$ovCoreWheelPath"
+
+          # Find and install the dev OV wheel
+          $ovDevWheelPath=Get-ChildItem -Path "${{ env.INSTALL_DIR }}\tools" -Filter openvino_dev*.whl | % { $_.FullName }
+          python3 -m pip install "$ovDevWheelPath[mxnet,caffe,kaldi,tensorflow2]"
+
+      - name: Install Python Layer tests dependencies
+        run: |
+          # For getting rid of SSL issues during model downloading for unit tests
+          python3 -m pip install certifi
+          # To enable pytest parallel features
+          python3 -m pip install pytest-xdist[psutil]
+          # layer test requirements
+          python3 -m pip install -r ${{ env.LAYER_TESTS_INSTALL_DIR }}/requirements.txt
+
+      - name: Set SSL_CERT_FILE for model downloading for unit tests
+        run: echo SSL_CERT_FILE=$(python3 -m certifi) >> $env:GITHUB_ENV
+
+      #
+      # Tests
+      #
+
+      - name: TensorFlow 1 Layer Tests - TF FE
+        if: fromJSON(inputs.affected-components).TF_FE.test
+        run: |
+          python3 -m pytest ${{ env.LAYER_TESTS_INSTALL_DIR }}/tensorflow_tests/ -n logical -m precommit  --junitxml=${{ env.INSTALL_TEST_DIR }}/TEST-tf_fe.xml
+        env:
+          PYTHONPATH: ${{ env.INSTALL_TEST_DIR }}/mo
```

Comment:
please add a comment here that this path needed for `unit-test` routines for layer tests.
I will remove this import in separate PR. This path is ideally not needed

Reply:
added